opencti-platform/opencti-front/src/private/components/common/stix_core_objects/StixCoreObjectFilesAndHistory.jsx

```diff
@@ -164,7 +164,7 @@ const StixCoreObjectFilesAndHistory = ({
     if (isCsvConnector && configuration && objectMarking) {
       const parsedConfig = JSON.parse(configuration);
       if (typeof parsedConfig === 'object') {
-        parsedConfig.user_chosen_markings = objectMarking.map((marking) => marking.value);
+        parsedConfig.markings = objectMarking.map((marking) => marking.value);
```

Comment:
why change this property name here ? I think it should stay user_chosen_markings